core/serializers.py

```diff
@@ -0,0 +1,44 @@
+from rest_framework import serializers
+from core.models import Tag, Source, SourceColumnMap, UpdateHistory
+from django.contrib.auth.models import User
+
+
+class TagSerializer(serializers.ModelSerializer):
+    user = serializers.ReadOnlyField(source='user.username')
+
+    class Meta:
+        model = Tag
+        fields = ('name', 'user', 'created_on', 'updated_on')
+
+
+class UserSerializer(serializers.ModelSerializer):
+    tag_set = TagSerializer(many=True, read_only=True)
+
+    class Meta:
+        model = User
+        fields = ('id', 'username', 'tag_set')
+
+
+class SourceColumnMapSerializer(serializers.ModelSerializer):
+    source = serializers.ReadOnlyField(source='source.indicator')
+
+    class Meta:
+        model = SourceColumnMap
+        fields = ('source', 'name', 'description', 'source_name')
+
+
+class UpdateHistorySerializer(serializers.ModelSerializer):
+    source = serializers.ReadOnlyField(source='source.indicator')
+
+    class Meta:
+        model = UpdateHistory
+        fields = ('source', 'history_table', 'is_major_release', 'released_on', 'release_description', 'invalidated_on', 'invalidation_description')
```

Comment:
Long line... could you move each option to it's own line. More readable. There's a JS/TS linter rule to keep the line length to max 120.